### ISSUE TITLE
Laog mignt be affected by a Fedora/RHEL systemd+selinux bug

### ISSUE THREAD

**Issue by ifireball** [lago-project/lago#111]
The following bug might affect Lago:
https://bugzilla.redhat.com/show_bug.cgi?id=1049656

It reproduces with libvirt+a recent RHEL7.2 cloud image. The recent addition of console support added to Lago might surface this.
Does Lago bring up VMs with selinux enabled ATM?

**Comment by david-caro** [comment]
Yes, but that only happens when creating the templates, if you are using the ones from the repos should be ok.
There's a workaround also for it, you have to add the following args to virt-builder/virt-sysprep/virt-customize:

   --edit '/usr/lib/systemd/system/fedora-autorelabel.service: $_ = "" if /StandardInput=tty/'

Or edit that file manually (see https://github.com/lago-project/lago-images/blob/initial_working_images/image-specs/el7-base for el* counterpart, just changes the name of the file)
